friends.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Developers Rising</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.100.1/css/materialize.min.css">
+  <link rel="stylesheet" href="./styles.css">
+  <link href="https://fonts.googleapis.com/css?family=Varela+Round" rel="stylesheet">
+</head>
+
+<body>
+
+  <div class="navbar-fixed">
+    <nav>
+      <div class="nav-wrapper grey">
+        <a href="./index.html" class="brand-logo">Developers Rising</a>
+        <ul id="nav-mobile" class="right hide-on-med-and-down">
+          <li><a href="https://www.meetup.com/DevelopersRising/">Events</a></li>
+          <li><a href="./friends.html">Team</a></li>
+          <li><a href="./members.html">Members</a></li>
+          <li><a href="./legal.html">Legal</a></li>
+          
+        </ul>
+      </div>
+    </nav>
+  </div>
+
+  <div class="section grey">
```

Comment:
Can you remove the contributors section, since this page should display cooperations and partners and not the members :)

Reply:
Done :+1